docs/release_notes/june_2024.md

```diff
@@ -0,0 +1,36 @@
+# June 2024
+
+## airbyte v0.62.3 to v0.63.4
+
+This page includes new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms.
+
+## ✨ Highlights
+
+Airbyte added [Refresh](operator-guides/refreshes) support for our BigQuery destination, which brings an improved experience to resyncing all your data again. This enables data to never be deleted from final tables during a historical resyncing of data. More certified destinations will be supported in the coming weeks. 
+
+Users can now monitor the progress of syncs through the addition of a live counter on the Connection Status page. The sync progress feature displays record counts and duration for each stream during a sync, which streams are syncing, and when a sync is not running. It also shows counts from the last time each enabled stream synced. Lastly, each stream now shows an individual stream status, which helps you troubleshoot by allowing you to pinpoint exactly which streams are erroring.
+
+![Sync Progress](./assets/sync-progress.png)
+
+## Platform Releases
+
+- (Cloud, Cloud Teams, and Self-Managed Enterprise only) Connections now display both the stream status and the latest records moved for recent syncs to aid with troubleshooting. 
+
+![Connection Graph](./assets/connection-stream-status-graph.png)
+
+- (Cloud Teams and Self-Managed Enterprise only) New roles have been released to support more defined access for larger teams. Editors (for Workspaces) and Readers (both Organization and Workspace) are now available. Read more in our [docs](/access-management/rbac).
+
+## Connector Improvements
```

Comment:
@natikgadzhi should we also write something about S3 being twice as fast?

Reply:
Added a suggestion below!

Reply:
Added!